diff --git a/arch/arm64/boot/dts/amlogic/meson-sm1-x96-max-plus.dts b/arch/arm64/boot/dts/amlogic/meson-sm1-x96-max-plus.dts
index eadf814b9..b56b88c98 100644
--- a/arch/arm64/boot/dts/amlogic/meson-sm1-x96-max-plus.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-sm1-x96-max-plus.dts
@@ -177,7 +177,7 @@ &uart_A {
 	uart-has-rtscts;
 
 	bluetooth: bluetooth {
-		compatible = "brcm,bcm43438-bt";
+		compatible = "realtek,rtl8822cs-bt";
 		interrupt-parent = <&gpio_intc>;
 		interrupts = <95 IRQ_TYPE_LEVEL_HIGH>;
 		interrupt-names = "host-wakeup";
